core/domain/email_manager.py

```diff
@@ -108,82 +113,54 @@
     }
 }
 
-NOTIFICATION_USER_IDS_LIST_SCHEMA: Dict[
-    str,
-    Union[
-        str,
-        Dict[str, Union[str, List[Dict[str, str]]]],
-        List[Dict[str, Union[str, int]]]
-    ]
-] = {
-    'type': schema_utils.SCHEMA_TYPE_LIST,
-    'items': {
-        'type': schema_utils.SCHEMA_TYPE_UNICODE,
-        'validators': [{
-            'id': 'is_valid_user_id',
-        }]
-    },
-    'validators': [{
-        'id': 'has_length_at_most',
-        'max_value': 5
-    }, {
-        'id': 'is_uniquified',
-    }]
-}
-
-EMAIL_HTML_BODY_SCHEMA: Dict[str, Union[str, Dict[str, int]]] = {
-    'type': schema_utils.SCHEMA_TYPE_UNICODE,
-    'ui_config': {
-        'rows': 20,
-    }
-}
-
-EMAIL_CONTENT_SCHEMA: Dict[
-    str,
-    Union[
-        str,
-        List[Dict[str, Union[str, Dict[str, Union[str, Dict[str, int]]]]]]
-    ]
-] = {
-    'type': schema_utils.SCHEMA_TYPE_DICT,
-    'properties': [{
-        'name': 'subject',
-        'schema': {
-            'type': schema_utils.SCHEMA_TYPE_UNICODE,
-        },
-    }, {
-        'name': 'html_body',
-        'schema': EMAIL_HTML_BODY_SCHEMA,
-    }],
-}
+EMAIL_SENDER_NAME: platform_parameter_domain.PlatformParameter = (
+    Registry.create_platform_parameter(
```

Comment:
Yeah, so I tried to list all of these platform parameters in the platform_parameter_list.py file where all of the platform params are listed but for some reason the tests were not able to locate the platform params and ended with thee following error:
```
======================================================================
ERROR: test_correct_email_draft_received_on_exploration_unpublish (core.domain.email_manager_test.ModeratorActionEmailsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ht/Desktop/oppia/oppia/core/domain/email_manager_test.py", line 6293, in test_correct_email_draft_received_on_exploration_unpublish
    d_text = email_manager.get_moderator_unpublish_exploration_email()
  File "/home/ht/Desktop/oppia/oppia/core/domain/email_manager.py", line 808, in get_moderator_unpublish_exploration_email
    platform_feature_services.get_platform_parameter_value(
  File "/home/ht/Desktop/oppia/oppia/core/domain/platform_feature_services.py", line 324, in get_platform_parameter_value
    raise PlatformParameterNotFoundException(
core.domain.platform_feature_services.PlatformParameterNotFoundException: Unknown platform parameter: UNPUBLISH_EXPLORATION_EMAIL_HTML_BODY.
```

Reply:
Ah, that's because the registry "collects" the params by examining the modules, I think. I guess if it works fine, we can leave as-is and see if we can optimize it later.